docs/dao/index.md

```diff
@@ -0,0 +1,43 @@
+---
+id: dao
+sidebar_label: JAM Implementers DAO
+sidebar_position: 5
+slug: /dao/
+---
+
+# JAM Implementers DAO 
+
+## Background 
+
+JAM Implementers DAO is a community of [JAM Implementers](https://jamcha.in/clients) dedicated to implementing JAM protocols, envisioned to underpin Polkadot’s technical evolution and the future of trustless supercomputing.  
+
+## Aims of the DAO
+
+By implementing JAM and engaging with the Polkadot broader Web3 + trustless supercomputing communities, we aim to advance the Polkadot/JAM  ecosystem and drive real-world adoption of Polkadot+JAM protocols.
+
+## Members
+
+This section functions as the single source of truth of JAM Implementer DAO Membership, supported by on-chain extrinsics.
+
+* [JAM DUNA](https://polkadot.subscan.io/extrinsic/25277967-2)	121Rs6fKm8nguHnvPfG1Cq3ctFuNAVZGRmghwkJwHpKxKjbx 
+* [Gossamer](https://polkadot.subscan.io/extrinsic/25278352-2) 143FppLAgb1KrvTRfkWKnozBeoZsZrhRr41gSBMyUDkUNT8z	
+* [Jamixir](https://polkadot.subscan.io/tx/0x2a66d68acab375502c60a4d36f168e476d7785189fcd916522a0a2666c746588)  15mXN5E3gn14t3etvcGHGfR4zL9WNoCBGCu9rJJiERqjePY8	
+* [JavaJAM](https://polkadot.subscan.io/extrinsic/25317339-2)	14zb7FpKGRCoW8SPezDHsMzqhGAuEn6Wjprf4j2Gn85dGVEh	
+* [JamZig ⚡️](https://polkadot.subscan.io/extrinsic/25319122-2)	14T1UcnoaMND24ubV86FCT4YxJ8q5CKVG7winwZ7umAit8g2	
```

Comment:
```suggestion
* [JamZig](https://polkadot.subscan.io/extrinsic/25319122-2)	14T1UcnoaMND24ubV86FCT4YxJ8q5CKVG7winwZ7umAit8g2	
```

The Remark only contained the name without the emoji.

Reply:
@ggwpez wanted to sign the remark with the emoji but anything out of the ASCII range is not accepted. I will be using the emoji everywhere else.

Reply:
Ok, lets leave it in then =)